src/centralwidget.cpp

```diff
@@ -50,8 +50,8 @@ void CentralWidget::setupWebView() {
             notification->setText(notificationInfo->message());
             notification->setPixmap(
                 QPixmap::fromImage(notificationInfo->icon()));
-            notification->setDefaultAction("View");
-            connect(notification, &KNotification::defaultActivated,
+            notification->addDefaultAction("View");
```

Comment:
This gives a warning since it's `nodiscard`, is there a way I can use the result of this method instead of `&KNotification::defaultActivated`? I don't know how to cpp 😛